cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -95,7 +96,16 @@ public static void setup() throws Exception {
         databaseLink.connection(c -> {
             DSLContext context = getDslContext(c, OFFICE_ID);
             OutletDao outletDao = new OutletDao(context);
+            LocationGroupDao locationGroupDao = new LocationGroupDao(context);
             try {
+                try {
+                    locationGroupDao.delete(NEW_RATED_OUTLET_CONTROLLED.getRatingCategoryId().getName(),
```

Comment:
This isn't doing what you think it's doing.  If the first delete fails, the second delete will be ignored.  You want to do a try/catch around each one.  Make it into a function like deleteLocation, so you don't muddy up the code.